.github/actions/spell-check/expect.txt

```diff
@@ -2165,21 +1720,16 @@ wcsnicmp
 WDA
 wdp
 wdupenv
-weakme
 webcam
 webpage
-websearch
-webserver
-website
+websites
 wekyb
 Wevtapi
+WEX
```

Comment:
This is a name of a team, I think. Why are we removing from names.txt and adding in expect.txt?